docs/releases/1.9.txt

```diff
@@ -163,6 +163,8 @@ Minor features
 
 * JavaScript slug generation now supports Romanian characters.
 
+* Added ``view`` permission for read only access to the models.
```

Comment:
This needs a more elaborate discussion of how the features works as well as links to the documentation added elsewhere.